src/main/java/com/github/dockerjava/api/command/CreateContainerCmd.java

```diff
@@ -417,6 +417,8 @@
 
     CreateContainerCmd withTty(Boolean tty);
 
+    CreateContainerCmd withTmpFs(Map<String, String> tmpFs);
```

Comment:
don't add hostconfig methods to create cmd. Fill hostconfig and set hostconfig in create